lib/messages.js

```diff
@@ -0,0 +1,254 @@
+/*
```

Comment:
All the code here is specific to slack. Should we rename this to lib/slack-messages.js?

Reply:
Derp, yes, that's a good idea.

Reply:
Done, Travis tests are running.